examples/word2vec/train_word2vec.py

```diff
@@ -117,28 +117,18 @@ def calculate_loss(model, dataset, position):
 if args.gpu >= 0:
     cuda.get_device(args.gpu).use()
 
-index2word = {}
-word2index = {}
-counts = collections.Counter()
-dataset = []
-with open('ptb.train.txt') as f:
-    for line in f:
-        for word in line.split():
-            if word not in word2index:
-                ind = len(word2index)
-                word2index[word] = ind
-                index2word[ind] = word
-            counts[word2index[word]] += 1
-            dataset.append(word2index[word])
-            if args.test and len(dataset) >= 100:
-                break
-        if args.test and len(dataset) >= 100:
-            break
-
-n_vocab = len(word2index)
+train, val, test = chainer.datasets.get_ptb_words()
```

Comment:
As a customary, we use `_` if this variable is never used. As `val` and `test` are not used afterwards, replace them with `_`.